src/tax/vertex-configure-address.md

```diff
@@ -1,36 +1,38 @@
 ---
-title: Configuring Storefront Address Validation
+title: Configuring Storefront Address Cleansing
 ---
 
-Your [Vertex Cloud account]({% link tax/vertex-account.md %}) also includes the Vertex Address Validation module. When this functionality is enabled for your Magento store, the storefront prompts the customer to correct the address information on both the shipping and billing steps of the one-page checkout and when the customer adds an address to their account.
+Your [Vertex Cloud account]({% link tax/vertex-account.md %}) also includes the Vertex Address Cleansing module. When this functionality is enabled for your Magento store, the storefront prompts the customer to correct the address information on both the shipping and billing steps of the one-page checkout and when the customer adds an address to their account.
```

Comment:
does this mean vertex compares the entered address against known post office addresses to populate a suggested address?